src/solidity/utils.ts

```diff
@@ -32,26 +32,35 @@ export function solidifyName(fileName: string): string {
 
 export async function gatherSources(
   what: string,
-  workingDir?: string,
+  workingDir: string = process.cwd(),
```

Comment:
@cymerrad @squadack The requirement for the resolver is its ability to be used in browser context - we can't depend on any Node packages that don't work in browsers. We need to remove all `process` and etc. occurrences and inject them from the client side

https://stackoverflow.com/questions/3058/what-is-inversion-of-control